README.md

```diff
@@ -26,7 +26,7 @@ wget https://github.com/risingwavelabs/risingwave/releases/download/v0.1.15/risi
 # Unzip the binary
 tar xvf risingwave-v0.1.15-x86_64-unknown-linux.tar.gz
 # Start RisingWave in single-binary playground mode
-./risingwave playground
+./risedev playground
```

Comment:
It's risingwave I guess (

Reply:
+1. We don't have risedev in the release.